packages/components/src/AccordionItem.astro

```diff
@@ -0,0 +1,21 @@
+---
+import "@gemini-ui-css/components/accordion.css";
+import NavArrowDownRegular from "@gemini-ui-astro/components/icons/regular/NavArrowDownRegular";
+
+interface Props {
+  title: string;
+  id: string;
```

Comment:
What is this for?

Reply:
At one point it was used to work out what should be open or not, I don't think its needed anymore